src/main/java/com/minecolonies/client/gui/WindowHutFarmer.java

```diff
@@ -1,198 +1,187 @@
 package com.minecolonies.client.gui;
 
+import com.blockout.Pane;
 import com.blockout.controls.Button;
 import com.blockout.controls.ButtonVanilla;
+import com.blockout.controls.ItemIcon;
+import com.blockout.controls.Label;
+import com.blockout.views.ScrollingList;
 import com.blockout.views.SwitchView;
 import com.minecolonies.MineColonies;
 import com.minecolonies.colony.buildings.BuildingFarmer;
+import com.minecolonies.entity.ai.citizen.farmer.FieldView;
 import com.minecolonies.lib.Constants;
-import com.minecolonies.network.messages.FarmerCropTypeMessage;
-import com.minecolonies.util.Log;
+import com.minecolonies.network.messages.AssignFieldMessage;
+import com.minecolonies.network.messages.AssignmentModeMessage;
+import com.minecolonies.network.messages.MinerSetLevelMessage;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.network.messages.MinerSetLevelMessage'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)